monolithe/generators/lang/java/templates/sdkinfo.java.tpl

```diff
@@ -0,0 +1,16 @@
+{{ header }}
+
+package {{ name }}.{{ version_string }};
+
+public interface SdkInfo {
+
+    double API_VERSION      = {{ version }};
+    String API_PREFIX       = "{{ api_prefix }}";
+    String PRODUCT_ACCRONYM = "{{ product_accronym }}";    
+    String PRODUCT_NAME     = "{{ product_name }}";    
+    String CLASS_PREFIX     = "{{ class_prefix }}";    
+    String NAME             = "{{ name }}";
+    String ROOT_API         = "{{root_api}}";
+    Class<?> ROOT_OBJECT_CLASS  = {{ class_prefix }}{{ root_api|capitalize }}.class;
```

Comment:
The template files used to produce sdk_info.py and SdkInfo.java use the same {{ variables }} so I am not sure where entity_name should be used.